docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -388,6 +388,62 @@ The default is `null`.
 Enables or disables P2P discovery.
 The default is `true`.
 
+### `ethstats`
+
+=== "Syntax"
+
+    ```bash
+    --ethstats=<nodename:secret@host:port>
```

Comment:
Are the ethstat options out of the experimental state yet? In the code, it still looks like it's `--Xethstats` and `--Xethstats-contact`